lienzo-core/src/main/java/com/ait/lienzo/client/core/shape/Shape.java

```diff
@@ -149,15 +150,15 @@ protected void setShapeType(final ShapeType type) {
     @Override
     public Map<ControlHandleType, IControlHandleList> getControlHandles(List<ControlHandleType> types) {
         if ((null == types) || (types.isEmpty())) {
-            return null;
+            return new HashMap<>();
```

Comment:
Hi @jomarko,

did you check all `getControlHandles` method invocations? If I correct some of them already relies on `null`s so this small change can lead to further issues.

Reply:
same here